.jshintrc

```diff
@@ -9,7 +9,7 @@
         "Promise",
```

Comment:
Maybe not a part of this patch, but jshint should be entirely removed. It's deprecated and was merged into ESLint, so we've moved into using ESLint in all our repos. You can see a simple repo like https://github.com/wikimedia/mediawiki-title to see the current best practice for linting.

Reply:
I would prefer to do it later. Recorded as T246012